.github/workflows/deploy_branches.yml

```
@@ -24,12 +24,13 @@ jobs:
           key: ${{ runner.os }}-modules-${{ hashFiles('./source/opam.export') }}
       - name: Install dependencies and build hazel # STEP 3
         run: |
-          sudo apt --assume-yes install curl m4 opam
+          sudo apt --assume-yes install curl m4 opam nodejs
           export OPAMYES=1
           opam init --compiler=ocaml-base-compiler.5.0.0
           eval $(opam env)
           make deps 
           make release
+          make test
```

Can we put this in another step so it's clear from the email if it fails?
```suggestion
      - name: Run tests
          make test
```

---

Outside the scope of this PR but placing here for a reminder that we should consider using
https://github.com/mirage/alcotest/issues/130 and https://github.com/marketplace/actions/test-reporter so we can get actual test report files.